src/index.html

```diff
@@ -8,6 +8,61 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__container">
+      <div class="stars stars--0">
+        <ul>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+      </div>
+      <div class="stars stars--1">
+        <ul>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+      </div>
+      <div class="stars  stars--2">
+        <ul>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+      </div>
+      <div class="stars  stars--3">
+        <ul>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+      </div>
+      <div class="stars  stars--4">
+        <ul>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+      </div>
```

Comment:
```suggestion
      <ul class="stars  stars--4">
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
      </ul>
```
Зачем этот div снаружи?